app/components/Game/Game.js

```diff
@@ -16,34 +19,71 @@ export default function Game({
     endButtonPressed,
     goButtonPressed,
   },
-  handleClicks,
+  handleClicks: {
+    handleClickOnBoard,
+    toggleStartButtonPressed,
+    toggleEndButtonPressed,
+    findAndDrawPath,
+  },
 }) {
-  const gameHeaderState = {
+  const instructionsState = {
     pathStartingPoint,
     pathEndingPoint,
     startButtonPressed,
     endButtonPressed,
   };
-  const gameScreenState = {
+  const instructionsProps = { instructionsState, };
+  const instructions = (
+    <div className="instructionsWrapper" key="2">
+      <GameInstructions { ...instructionsProps } />
+    </div>
+  );
+  const settingsText = '<-- Settings';
+  const linkProps = {
```

Comment:
settingsLinkProps, as these are the props of a very specific link

Reply:
done